app/src/main/java/com/davidtomas/taskyapp/features/agenda/presentation/AgendaDetail/AgendaDetailRoot.kt

```diff
@@ -45,6 +51,13 @@ fun AgendaDetailRoot(
                 }
             }
         }
+    }
+    LaunchedEffect(key1 = isPhotoDeleted) {
+        isPhotoDeleted?.value?.let { isPhotoDeleted ->
+            if (isPhotoDeleted) agendaDetailViewModel.populateData()
```

Comment:
What will populateData() do here? Doesn't sound like it removes a photo from the list 🤔